app/policies/case/sar/offender_policy.rb

```diff
@@ -55,9 +55,10 @@ def can_send_dispatch_letter?
     check_user_can_manage_offender_sar
   end
 
-  def handle_information_received?
+  def can_validate_rejected_case?
     clear_failed_checks
-    check_user_can_manage_offender
+    edit_case?
```

Comment:
Do you need to check this together with the next line?
If `edit_case?` returns false and `check_can_trigger_event(:validate_rejected_case)` returns true then `can_validate_rejected_case?` will still return true.